### ISSUE TITLE
May I ask if the phase change module can be coupled to the ultrasound module for HIFU simulation?

### ISSUE THREAD

**Issue by Kolihanar** [MFlowCode/MFC#658]
Very outstanding work. We are currently searching for a solution that can combine high-intensity focused ultrasound to generate bubbles. Of course, this phenomenon involves many phase transition processes. I noticed that the program includes m_phase-change.fpp. Can we now consider combining the ultrasound module with the m_phase-change.fpp module? At present, the only phase transition process in the example case is in the shock tube, so I would like to ask if it is possible for developers to combine HIFU and m_phase-change.fpp with existing code?

**Comment by sbryngelson** [comment]
Yes you can. If you have a more specific question, you can ask on our Slack. If there's an issue with MFC, let us know here!
